Software/Development/SemesterProject/App/analogAcquisitionManager.c

```diff
@@ -359,9 +366,9 @@ void createAnalogSignals(void)
      * Then pass the maximum and then the minimum value of the threshold you want to set.
      */
     setCurrentThresholds(&currentThreshold[0], 250, -250);
-    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 45, 25);
+    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 45, 0);
```

Comment:
I understand why you did that. Maybe we should list somewhere all the changes to undo for the final system?